chainer/training/extensions/progress_bar.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import division
+import datetime
+import sys
+import time
+
+from chainer.training import extension
+from chainer.training import trigger
+
+
+class ProgressBar(extension.Extension):
+
+    """Trainer extension to print a progress bar and recent training status.
+
+    This extension prints a progress bar at every call. It watches the current
+    iteration and epoch to print the bar.
+
+    Args:
+        training_length (tuple): Length of whole training. It consists of an
```

Comment:
I meant this corresponds to the epoch count or iteration count at a full progress bar. The unit is also specified in this argument, thus I just wrote "length", but I agree that it is not clear.